docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,629 @@
+---
+title: Submitting a pull request to Loopback 4
+lang: en
+keywords: contributing, LoopBack community, pull request, PR, loopback
+sidebar: contrib_sidebar
+permalink: /doc/en/lb4/submitting_a_pr.html
+---
+
+## How to submit a pull request (PR) for Loopback 4
+
```

Comment:
since there are quite a number of steps, I wonder if we should have an index for the steps at the top of the page, something similar to https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md#developing-loopback.

Reply:
I believe our doc site generator puts an auto-generated table-of-contents at the top of every page.

In `DEVELOPING.md`, I turned off this auto-generated TOC because I want the file to be readable on GitHub too.

I don't expect people to read `submitting_a_pr` directly on GitHub, therefore I am ok to use the auto-generated TOC that's displayed on `loopback.io` only.

Reply:
this is the toc at the top of the page

![image](https://user-images.githubusercontent.com/6864736/52658424-f88f7980-2ec8-11e9-9272-0b7aec3fbfba.png)